src/components/content/Content.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <div class="content">
+    <CardBox v-for="card in stateController.card" :card="card" />
+  </div>
+</template>
+
+<script>
+import { useCardStore } from '../../stores/counter';
+import CardBox from '../card/CardBox.vue';
+
+export default {
+  name: 'Content',
+  setup() {
+    const stateController = useCardStore();
+    return { stateController };
+  },
+  components: { CardBox },
+  methods: {
+    getLocalData() {
+      if (localStorage.getItem('LocalData')) {
+        this.stateController.getData(
```

Comment:
why it's called getData if  it is setting data?

Reply:
i changed name to checkLocalData because i have if else condition

Reply:
@HrachYaralyan this function is not getting any data. Change the name